frontend/src/app/pages/exercises/exercise/shared/trainer-map-editor/trainer-map-editor.component.html

```diff
@@ -136,6 +136,18 @@ <h5>
                             height: 240px;
                         "
                     >
+                        <label class="btn btn-outline-primary ms-5">
```

Comment:
This input is below the other buttons and texts in this row:
![image](https://user-images.githubusercontent.com/68013019/181906546-87a3576e-c0c8-4728-a137-a41bb58c2d6b.png)

This breaks the whole layout for me when there are more patients:
![image](https://user-images.githubusercontent.com/68013019/181906655-8cb1e2c5-aa6c-4e5e-84c2-a77a799488a2.png)

Reply:
Now the button doesn't work at all. And I'm not sure I like that the text is less descriptive now.